### ISSUE TITLE
Screen Reader redundantly announcing header and tab items in Link plugin button

### ISSUE THREAD

**Issue by lumills** [ckeditor/ckeditor4#4051]
## Type of report

Bug

## Provide detailed reproduction steps (if any)
1) Ckeditor4 instance with Link plugin and toolbar button

2)Navigate to "Editor Tool Bar" section.

3)Navigate to Button "Link" and click on it.

4)Navigate To tab panel "Advanced" tab item option.

5)Start Narrator and navigate whole content.

### Expected result

​It Should not announce “heading level 1 Link” and "Advance" again and again. It should announce only selected content.

### Actual result

Narrator redundantly announces heading and tab item name again and again rather than just once.

## Other details

* Browser: Firefox (repro in others as well)
* CKEditor version: 4

**Comment by f1ames** [comment]
Hi @lumills,

based on "Actual result" part of your description:

> Narrator redundantly announces heading...

I assume this issue occurs when using Narrator? Is it the only screen reader you have encountered checked or are there more where you have encountered this issue?

**Comment by lumills** [comment]
Hi @f1ames ,

As per the bug given to me it only claims Narrator to be producing this issue.

**Comment by Comandeer** [comment]
I can't reproduce it, the tab name is announced only for the first item in the tab. Then only the name of the selected field is announced.
